src/ex6/server/client/src/index.css

```diff
@@ -0,0 +1,16 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```

Comment:
these are a lot of fonts, they are not all useful- use only what you need
also you override these anyway further down the line in .app-container
so this can be removed - I assume